### PR TITLE
[CAR] Added entity, added table in liquibase, added car repository

### PR DESCRIPTION
**Pull request by AntonZhdanov**

### CODE REVIEW

File: src/main/java/com/project/carsharingapp/model/Car.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+@Entity
+@SQLDelete(sql = "UPDATE cars SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted=false")
+@Table(name = "cars")
+public class Car {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String model;
+    private String brand;
```
Comment (DmytroV95) [inline src/main/java/com/project/carsharingapp/model/Car.java]: Add  @Column(nullable = false) for model and brand

File: src/main/java/com/project/carsharingapp/model/Car.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import java.math.BigDecimal;
+import javax.validation.constraints.PositiveOrZero;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Getter
+@Setter
+@Entity
```
Comment (DmytroV95) [inline src/main/java/com/project/carsharingapp/model/Car.java]: Replace annotation Entity on the top, before setter and getter

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -3,6 +3,7 @@ spring.datasource.username=root
 spring.datasource.password=gjujlf123$_0
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 
+
```
Comment (DmytroV95) [inline src/main/resources/application.properties:5]: Redundant line

**Review by DmytroV95 — CHANGES_REQUESTED** [review]
Added some comments to impore.

File: src/main/java/com/project/carsharingapp/model/Car.java (hunk 1 above)
Comment (ivan0dyatlyukkk) [reply src/main/java/com/project/carsharingapp/model/Car.java]: @DmytroV95 @AntonZhdanov I guess that the daily fee should be not null as well, what're your opinions?

File: src/main/java/com/project/carsharingapp/model/Car.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String model;
+    private String brand;
+    @PositiveOrZero
+    private Integer inventory;
+    @Enumerated(value = EnumType.STRING)
+    private CarType carType;
+    @PositiveOrZero
+    private BigDecimal dailyFree;
```
Comment (ivan0dyatlyukkk) [inline src/main/java/com/project/carsharingapp/model/Car.java]: Please, rename to dailyFee:)
```
@Column(name = "daily_fee", nullable = false)
private BigDecimal dailyFee
```

**Review by ivan0dyatlyukkk — CHANGES_REQUESTED** [review]

**Review by DmytroV95 — APPROVED** [review]
GJ!

**Review by ivan0dyatlyukkk — APPROVED** [review]

**Pull request merged**